stepactions/reduce-snapshot/reduce-snapshot.yaml

```diff
@@ -0,0 +1,88 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: StepAction
+metadata:
+  name: reduce-snapshot-action
+spec:
+  image: quay.io/konflux-ci/appstudio-utils:ab6b0b8e40e440158e7288c73aff1cf83a2cc8a9@sha256:24179f0efd06c65d16868c2d7eb82573cce8e43533de6cea14fec3b7446e0b14
```

Comment:
It looks like you are only using oc and jq. Can you just use the release-service-utils image with kubectl in place of oc so that we don't have to worry about who maintains this image?

Reply:
changed the image.